DOCKER.md

```diff
@@ -0,0 +1,22 @@
+Docker
+-------------
+
+Running Nukkit in [Docker](https://www.docker.com/) (17.05+ or higher).
```

Comment:
This should read `Cloudburst Server` instead of `Nukkit`

Reply:
I hadn’t changed that but in case it wouldn’t work